manifests/klusterlet/managed/klusterlet-work-clusterrolebinding-execution.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
-  name: open-cluster-management:{{ .KlusterletName }}-work:execution
+  name: open-cluster-management:{{ .KlusterletName }}-work:aggregate
```

Comment:
since we have the aggregate rule, do we still need this binding?

Reply:
yes, we did